### PR TITLE
Jetpack Manage: Remove mock content and update content sidebar styles

### PR DESCRIPTION
**Pull request by cleacos**
Resolve https://github.com/Automattic/jetpack-manage/issues/177

## Proposed Changes

This PR removes mock content and updates the content-sidebar styles to fit with the design, applying the wideLayout style.

image

## Testing Instructions

- Check the code and styles applied.
- Check that the content appears with a card placeholder (WIP) and the right sidebar.
- On large horizontal screens, the content and the sidebar should be in the center at a fixed width (wide layout).
- Check that it is responsive and below 1080px the sidebar goes at the bottom and the content at the top.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/jetpack-cloud/components/content-sidebar/style.scss [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
 		display: grid;
-		grid-template-columns: 3fr 1fr;
-		column-gap: 32px;
-		min-height: calc(100vh - 123px);
-		max-width: 1500px;
+		grid-template-columns: repeat(12, [col-start] minmax(0, 1fr));
+		grid-gap: 32px;
+		grid-column-gap: 24px;
+	}
+	.content-sidebar__main-content {
```
Comment (tbradsha) [inline client/jetpack-cloud/components/content-sidebar/style.scss]:
```suggestion
	&__main-content {
```

File: client/jetpack-cloud/components/content-sidebar/style.scss [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
+		@include break-xlarge {
+			grid-column: 1 / span 8;
+		}
 	}
 
-	.right-sidebar {
-
-		> *:not(:first-child) {
-			margin-top: 40px;
+	.content-sidebar__right-sidebar {
```
Comment (tbradsha) [inline client/jetpack-cloud/components/content-sidebar/style.scss]:
```suggestion
	&__right-sidebar {
```

File: client/jetpack-cloud/sections/overview/primary/sidebar/style.scss [hunk 3]
```diff
@@ -1,8 +1,12 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 
-.jetpack-cloud-content-sidebar {
-	.right-sidebar {
+.content-sidebar {
+	.content-sidebar__right-sidebar {
```
Comment (tbradsha) [inline client/jetpack-cloud/sections/overview/primary/sidebar/style.scss]:
```suggestion
	&__right-sidebar {
```

**Comment by tbradsha** [comment]
I added a few inline suggestions to clean up the SASS (non-blockers). I do not like how the main content floats right instead of left; is that something design suggested? This is particularly annoying since the left gap is visible on my own standard laptop screen.

I briefly poked at the merge conflict, but had some issues so backed out.

File: client/jetpack-cloud/sections/overview/primary/sidebar/style.scss (hunk 3 above)
Comment (cleacos) [reply client/jetpack-cloud/sections/overview/primary/sidebar/style.scss]: https://github.com/Automattic/wp-calypso/pull/85366/commits/a9e36d119421fdd9422a52556635d2f6ede357dd

File: client/jetpack-cloud/components/content-sidebar/style.scss (hunk 1 above)
Comment (cleacos) [reply client/jetpack-cloud/components/content-sidebar/style.scss]: https://github.com/Automattic/wp-calypso/pull/85366/commits/a9e36d119421fdd9422a52556635d2f6ede357dd

File: client/jetpack-cloud/components/content-sidebar/style.scss (hunk 2 above)
Comment (cleacos) [reply client/jetpack-cloud/components/content-sidebar/style.scss]: https://github.com/Automattic/wp-calypso/pull/85366/commits/a9e36d119421fdd9422a52556635d2f6ede357dd

**Review by tbradsha — APPROVED** [review]
Matches WP.com Calypso layout. Approved.

**Pull request merged**